src/imitation/algorithms/mce_irl.py

```diff
@@ -234,7 +232,8 @@ def predict(  # type: ignore[override]
                 actions.append(self.rng.choice(len(dist), p=dist))
 
         timesteps += 1  # increment timestep
-        return np.array(actions), timesteps
+        state = (timesteps,)
```

Comment:
This is a breaking change in the API. I updated the tests to reflect the new usage (have to extract the zeroth element from state). What is the reason behind this? We always return a tuple of length 1 anyways. Is this for compatibility with other objects / functions?

Reply:
sounds good. Marking this as resolved then.